ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -29,10 +32,15 @@ def client_interceptor(tracer):
     """
     from . import _client
 
+    if tracer_provider is None:
+        tracer_provider = trace.get_tracer_provider()
+
+    tracer = tracer_provider.get_tracer(__name__, __version__)
```

Comment:
Since this pattern will occur more often, would it make sense to add an utility function `setup_tracer(tracer_provider: Optional[TracerProvider], name, version)` to the API to encapsulate these three lines?

Reply:
I was thinking if it could make sense to add an optional `TracerProvider` to `get_tracer()`. It'll be the same `setup_tracer` you're proposing.